### PR TITLE
Submitting governance actions: Use latest guardrails script

### PR DESCRIPTION
**Pull request by CarlosLopezDeLara**
👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.

1. Please complete a Checklist
2. Fill in all sections of the template
3. Click "Create pull request"

## Checklist

 <-- Please fill the boxes with [x] before submitting a pull request --> 

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 
- [x] I have not committed any changes to `yarn.lock` (or have [removed these changes](https://developers.cardano.org/docs/portal-contribute/#i-committed-yarnlock-to-my-pr-branch-what-do-i-do)).

## Updating documentation or Bugfix

<-- Help us understand your motivation by explaining why you decided to make this change. We must be able to understand the purpose of your change from this description.  Does this fix a bug? Does it close an issue? If so please mention it.  -->

### CODE REVIEW

**Review by disassembler — APPROVED** [review]

File: docs/get-started/cardano-cli/governance/governance-actions.md [hunk 1]
```diff
@@ -196,25 +196,25 @@ At the Chang hardfork, the interim constitution will be supplemented with a Guar
 
 The guardrails script applies only to protocol parameter update and treasury withdrawal proposals.
 
-Follow the instructions in the README.md file to compile the PlutusV3 script. A successful compilation creates the 'compiled' directory containing the script in a text envelope.
+Follow the instructions in the README.md file to compile the PlutusV3 script. A successful compilation creates the follwoing:
```
Comment (rphair) [inline docs/get-started/cardano-cli/governance/governance-actions.md:199]:
```suggestion
Follow the instructions in the README.md file to compile the PlutusV3 script. A successful compilation creates the following:
```

**Review by rphair — COMMENTED** [review]
thanks @disassembler for double-checking.

**Review by rphair — APPROVED** [review]

**Review by rdlrt — APPROVED** [review]

**Review by katomm — APPROVED** [review]

**Comment by katomm** [comment]
I believe we are all in an agreement that based on contributions and reputation that @disassembler and @CarlosLopezDeLara have been added as a collaborator with write permissions on the repository so that their review counts as approval going forward.

**Comment by CarlosLopezDeLara** [comment]
@katomm I'm a bit puzzled. This is merged but I don't see the changes on the live site.

**Comment by rdlrt** [comment]
Merged to staging, not main branch yet

**Comment by CarlosLopezDeLara** [comment]
> Merged to staging, not main branch yet

I suppose this is automated. Is there a way for me to track it?

**Comment by katomm** [comment]
It is not automated. We often push to main Fridays, also today. It is live now.

Staging is on https://staging-dev-portal.netlify.app/ btw.

**Comment by katomm** [comment]
Note: We should consider doing releases for the developer portal like on cardano.org https://github.com/cardano-foundation/cardano-org/releases in the future.

**Pull request merged**